.github/workflows/golang.yaml

```diff
@@ -0,0 +1,27 @@
+on: [push]
+
+jobs:
+  golang:
+    runs-on: ubuntu-latest # we execute everything except make in docker anyway
```

Comment:
Shouldn't we pin this to an LTS ubuntu?

Reply:
https://github.com/actions/runner-images/tree/main/images/ubuntu latest = LTS. I was surprised to see this too :)

Reply:
oh ok cool